packages/webapp/pages/squads/[handle]/edit.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { ReactElement } from 'react';
+import { NextSeo, NextSeoProps } from 'next-seo';
+import router, { useRouter } from 'next/router';
+import { useAuthContext } from '@dailydotdev/shared/src/contexts/AuthContext';
+import Unauthorized from '@dailydotdev/shared/src/components/errors/Unauthorized';
+import { SquadDetails } from '@dailydotdev/shared/src/components/squads/Details';
+import { SquadForm, editSquad } from '@dailydotdev/shared/src/graphql/squads';
+import { useBoot } from '@dailydotdev/shared/src/hooks/useBoot';
+import { useToastNotification } from '@dailydotdev/shared/src/hooks/useToastNotification';
+import {
+  ManageSquadPageContainer,
+  ManageSquadPageMain,
+  ManageSquadPageHeader,
+} from '@dailydotdev/shared/src/components/squads/utils';
+import { MangeSquadPageSkeleton } from '@dailydotdev/shared/src/components/squads/MangeSquadPageSkeleton';
+import { useQueryClient } from 'react-query';
+import { useSquad } from '@dailydotdev/shared/src/hooks';
+import {
+  GetStaticPathsResult,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+} from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import {
+  Button,
+  ButtonSize,
+} from '@dailydotdev/shared/src/components/buttons/Button';
+import ArrowIcon from '@dailydotdev/shared/src/components/icons/Arrow';
+import { defaultOpenGraph, defaultSeo } from '../../../next-seo';
+import { getLayout as getMainLayout } from '../../../components/layouts/MainLayout';
+
+type EditSquadPageProps = { handle: string };
+
+const seo: NextSeoProps = {
+  title: 'Create your Squad',
```

Comment:
This should be different, maybe check quickly what the old modal title was.

Reply:
There was no title
<img width="509" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/026537ca-104c-49ad-ade0-57fe27ad78be">

Reply:
Just `Create squad` should do?

Reply:
`Squads early access` seems weird since you are on squad create page.

Reply:
It's the edit page right?
So should be something in line of Edit your squad

Reply:
Oh sorry, I somehow seen it on create, it was `Squad settings` on modal. I'll update.